### ISSUE TITLE
Cannot pass PostToConnectionCommand instance to ApiGatewayManagementApiClient's send() function as argument

### ISSUE THREAD

**Issue by OrkhanGG** [aws/aws-sdk-js-v3#6442]
### Describe the issue

Argument type PostToConnectionCommand is not assignable to parameter type
Command<ServiceInputTypes, SI, ServiceOutputTypes, SO, SmithyResolvedConfiguration<__HttpHandlerOptions>>
   Type
(stack: IMiddlewareStack<any, any>, configuration: {     logger: Logger,     requestHandler: RequestHandler<any, any, any> }, options: any) => Handler<SI, SO>
is not assignable to type
(stack: MiddlewareStack<ServiceInputTypes, ServiceOutputTypes>, configuration: SmithyResolvedConfiguration<__HttpHandlerOptions>, options: any) => Handler<SI, SO>

### Links

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/apigatewaymanagementapi/command/PostToConnectionCommand/

**Comment by aBurmeseDev** [comment]
HI @OrkhanGG - thanks for reaching out. 

Can you please elaborate more on the behavior with your code example and version used? I'm able to run it successfully with code below.

```js
import { ApiGatewayManagementApiClient, PostToConnectionCommand } from "@aws-sdk/client-apigatewaymanagementapi"; // ES Modules import
const client = new ApiGatewayManagementApiClient(config);
const input = { /
  Data: new Uint8Array(), 
  ConnectionId: "STRING_VALUE", 
};
const command = new PostToConnectionCommand(input);
const response = await client.send(command);
```
Best,
John

**Comment by OrkhanGG** [comment]
Hi @aBurmeseDev!

I encountered issues using the official `javascript v3` SDK with AWS Lambda, specifically facing timeouts without any explicit error messages. Even without additional processing, my goal was simply to send a message to the client via AWS WebSockets API Gateway.

After some troubleshooting, I realized that AWS Lambda already includes the necessary modules for `@aws-sdk/client-apigatewaymanagementapi`, so there was no need to install it separately (something I initially overlooked). Once I uninstalled the relevant node modules and leveraged the built-in support, everything worked as expected.

For those who encounter the same problem, here's the code that worked for me, without requiring additional credentials. I also ensured the Lambda function had the appropriate permissions to access the API Gateway:

```javascript
import { ApiGatewayManagementApiClient, PostToConnectionCommand } from "@aws-sdk/client-apigatewaymanagementapi";

const client = new ApiGatewayManagementApiClient({ endpoint: process.env.API_GATEWAY_ENDPOINT });

const publish = async (connectionId, message) => {
    console.log('Connection ID and message:', connectionId, message);
    
    const requestParams = {
        ConnectionId: connectionId,
        Data: JSON.stringify({ message })
    };
    
    const command = new PostToConnectionCommand(requestParams);
    
    try {
        await client.send(command);
    } catch (error) {
        console.error('Error sending message:', error);
    }
    
    return { statusCode: 200 };
}

export default publish;
```

This solution eliminated the timeouts and allowed my Lambda function to send WebSocket messages reliably. 

@aBurmeseDev, it would be greatly appreciated if the documentation were clearer and more straightforward. The docs state `const client = new ApiGatewayManagementApiClient(config);`, but they don’t provide sufficient detail on what the `config` should ideally include. There are several key aspects missing, and I struggled to find up-to-date examples online.

I hope this helps anyone encountering a similar issue!
